lib/git_reflow/commands/refresh.rb

```diff
@@ -0,0 +1,26 @@
+desc 'refresh will rebase your current branch based on the base branch'
+long_desc <<LONGTIME
+  Performs the following:\n
+  \t$ git checkout <base_branch>\n
+  \t$ git pull --rebase <remote_location> <base_branch>\n
+  \t$ git checkout <current_branch>\n
+  \t$ git rebase <base_branch>\n
+LONGTIME
+arg_name '[base_branch] - branch that you want to rebase on (master by default), [remote_location] - remote repository name to fetch updates from (origin by default)'
+command :refresh do |c|
+  c.action do |global_options, options, args|
+    if args.empty?
+      raise "usage: git reflow refresh [base_branch_name - defaults to master] [remote_location - defaults to origin]"
+    else
+      # defaults remote_location to origin
+      remote_location = args.length < 2 ? "origin" : args[1]
+      base_branch = args.length < 1 ? "master" : args[0]
+      current_branch = GitReflow.current_branch
+
+      `git checkout #{base_branch}`
+      `git pull --rebase #{remote_location} #{base_branch}`
```

Comment:
Ok, I was just following the code sample from `git reflow start`. Thanks for pointing me to `update_destination(base_branch)`, its a great idea to implement such a method in `git_reflow.rb`!